bin/kano-settings-onboot

```diff
@@ -217,6 +222,26 @@ def ensure_correct_volume_levels(dry_run=False):
     # return info
 
 
+def ensure_screen_kit_rotation_global():
+    """
+    Ensure the screen kit orientation setting is also set globally.
+
+    This simply mirrors the display_rotate option for the current SK to the
+    global space for the unlikely case where an SK loses its EDID.
+    """
+    was_set_once = os.path.exists(DISPLAY_ROTATE_GLOBAL_FLAG)
+    if was_set_once:
+        return
+
+    if is_screen_kit():
+        rotation = get_screen_value('display_rotate')
+        set_config_value('display_rotate', rotation)
+
+        if not os.path.exists(SYSTEM_FLAGS_DIR):
+            os.makedirs(os.path.dirname(SYSTEM_FLAGS_DIR))
+        open(DISPLAY_ROTATE_GLOBAL_FLAG, 'a').close()
```

Comment:
Maybe wrap in a try in case permissions or something causes this to traceback and not run the rest of the script.

Reply:
Wrapped the whole function 👍